.circleci/config.yml

```diff
@@ -8,6 +8,8 @@ jobs:
         executor:
```

Comment:
Not making any suggestions, this is genuine curiosity! Is there a reason why we're using circleci instead of github actions? Or are they roughly equivalent and we just picked one.

Reply:
There are reasons, but they are a little weak. I'm totally open to a conversation if you become convinced that switching makes sense. My reasons are roughly:
- I am personally already familiar with CircleCI, so it was quick and easy to get launched.
- It's generally nice for the whole team to have consistent tooling across projects (I'm including other python snake-charmers projects here).
- I have some resistance to Github Lock-in, though they've already got their claws in us pretty deep.
- I have no major concerns about Circle and know of no superiority of Github Actions, so didn't feel inspired to spend time learning the Github Actions way of doing things, even if it's roughly equivalent or marginally faster.